Makefile

```diff
@@ -3,6 +3,7 @@
 OPENAPI_GENERATOR_VERSION=5.0.1
 
 install:
+	cd examples/go && go get github.com/twilio/twilio-go@main
```

Comment:
Not something to fix right now, but don't like that we're requiring `go` during install since we may use this repo to generate other languages in the future.

Reply:
Agreed. The goal here is to pull the latest commit from twilio-go when running tests. Would it be better to move this into the `./prism.sh` file?

Reply:
I'm okay with it here for now since it makes working with the Go tests in an IDE better.